streaming-binlog/src/main/java/org/openmrs/analytics/FhirUriGenerator.java

```diff
@@ -22,22 +22,24 @@
 import org.slf4j.Logger;
 import org.slf4j.LoggerFactory;
 
-/**
- * A Debezium events to FHIR URI mapper
- * can be configured to use the utils/fhir2_atom_feed_config.json mapper
- */
+
+ // A Debezium events to FHIR URI mapper
 public class FhirUriGenerator implements Processor {
 
-    private static final Logger LOG = LoggerFactory.getLogger(DebeziumListener.class);
+    private static final Logger log = LoggerFactory.getLogger(DebeziumListener.class);
```

Comment:
`DebeziumListener.class` -> `FhirUriGenerator.class`

Reply:
nice catch I'll fix this in the next PR

Reply:
I suppose you mean next commit in this PR, right? :-)

Reply:
yea next commit :)